label_sync/README.md

```diff
@@ -61,12 +61,14 @@ bazel run //label_sync -- \
   --only kubernetes/community,kubernetes/steering
   # see above
 
-# generate docs based on labels.yaml
+# generate docs and a css file contains labels styling based on labels.yaml
 bazel run //label_sync -- \
   --action docs \
   --config $(pwd)/label_sync/labels.yaml \
   --docs-template $(pwd)/label_sync/labels.md.tmpl \
-  --docs-output $(pwd)/label_sync/labels.md
+  --docs-output $(pwd)/label_sync/labels.md \
+  --css-template $(pwd)/label_sync/labels.css.tmpl \
+  --css-output $(pwd)/label_sync/labels.css
```

Comment:
we also need these flags updated in:
https://github.com/kubernetes/test-infra/blob/69d3140896e9761deed0f173cbfb22d76db34e8b/hack/update-labels.sh#L25-L30, the presubmit will continue to fail otherwise